assets/models/system/Mixtral-8x7B-v0-1/spec.yaml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mistralai-Mixtral-8x7B-v01
+path: ./
+properties:
+  inference-recommended-sku: Standard_NC24s_v3, Standard_NC24rs_v3,
+    Standard_ND40rs_v2, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4    
+  languages: EN
+tags:
+  Featured: ''
+  SharedComputeCapacityEnabled: ''
```

Comment:
Discussed offline.
Batch isn't tested. Add `disable-batch : true` for batch not supported case